ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/checkboxgroup/v1/checkboxgroup/checkboxgroup.html

```diff
@@ -36,14 +37,16 @@
     <div class="cmp-adaptiveform-checkboxgroup__widget ${checkboxgroup.orientation}">
         <div data-sly-list="${checkboxgroup.enums}" data-sly-unwrap>
             <div class="cmp-adaptiveform-checkboxgroup-item ${checkboxgroup.name}">
-                <label class="cmp-adaptiveform-checkbox__label" aria-label="${checkboxgroup.enumNames[itemList.index]}"
+                <label class="cmp-adaptiveform-checkbox__label" 
+                       for="${component.id}${'_'}${itemList.index}__widget"
                        title="${checkboxgroup.tooltipVisible ? '' : checkboxgroup.tooltipText}"
-                       for="${component.id}${'_'}${itemList.index}__widget">
+                       aria-label="${checkboxgroup.enumNames[itemList.index]}"
+                       >
                     <input class="cmp-adaptiveform-checkbox__widget" type="checkbox" id="${component.id}${'_'}${itemList.index}__widget"
                            aria-describedby="${checkboxgroup.labelForId}_desc"
                            name="${checkboxgroup.name}"
                            value="${checkboxgroup.enums[itemList.index].toString}"
-                           checked="${checkboxgroup.enums[itemList.index] in checkboxgroup.default }"
+                           aria-checked="${checkboxgroup.enums[itemList.index] in checkboxgroup.default }"
```

Comment:
This is native checkbox, does it really need aria-checked attribute? Shouldn't checked attribute be sufficient?